src/components/Dialog.tsx

```diff
@@ -104,12 +104,13 @@ export function Header({ title }: HeaderProps) {
 
 export const Modal = styled.div<{ color: Color }>`
   background-color: ${({ color, theme }) => theme[color]};
-  border-radius: ${({ theme }) => theme.borderRadius * 0.75}em;
+  border-radius: ${({ theme }) => theme.borderRadius}em;
```

Comment:
The only dialog that still has the old-style borders in figma are transaction error dialogs, I'm checking with fred on whether that's intentional or if it can be removed from all.

Reply:
Fred has confirmed all dialogs should be changed 👍